package.json

```diff
@@ -71,12 +73,12 @@
     "@babel/plugin-proposal-pipeline-operator": "^7.18.9",
     "@babel/preset-env": "^7.18.9",
     "@babel/preset-react": "^7.18.6",
-    "@storybook/addon-a11y": "6.4.22",
-    "@storybook/addon-storysource": "6.4.22",
-    "@storybook/addon-toolbars": "6.4.22",
-    "@storybook/addons": "6.4.22",
-    "@storybook/react": "6.4.22",
-    "@storybook/source-loader": "6.4.22",
+    "@storybook/addon-a11y": "6.5.10",
+    "@storybook/addon-storysource": "6.5.10",
+    "@storybook/addon-toolbars": "6.5.10",
+    "@storybook/addons": "6.5.10",
+    "@storybook/react": "6.5.10",
+    "@storybook/source-loader": "6.5.10",
     "@testing-library/dom": "8.14.0",
```

Comment:
I ran into the wrap with act issue in the tests previously when I tried to upgrade awhile ago. I went through and upgraded the testing dependencies, which solved the wrapped in act issue, but resulted in different testing errors.

Note that looking through the testing dependencies a lot of them are frozen at a specific version, we should hopefully be able to move most of them up now

Reply:
I updated the testing-library/dom to ^8.17.1 and @testing-library/user-event to ^14.4.3. The only failing tests I have at the moment are the List disabled/reorder one that others saw I think and a SelectMultiple keyboard interaction one. Still looking into the details.

Reply:
It feels like we shouldn't need to wrap user events in act. I found this thread on the wrapping with act issue coming up when upgrading to react 18: https://github.com/testing-library/user-event/discussions/906

Reply:
Very true. Thanks! Looks like the upgrades cleared that up.